### DISCUSSION TITLE
Sdm120 two energy meter HELP ?

### DISCUSSION THREAD

**Discussion by natesty2022** [arendst/Tasmota#15058] [Q&A] [answered]
How add two sdm120 to one rs485 converter in tasmota ???

**Comment by barbudor** [comment, marked as answer]
The native SDM120 driver is designed to support only 1 SDM120 per ESP !!!

Maybe you could do it using Smart-Meter script : https://tasmota.github.io/docs/Smart-Meter-Interface/

**Reply by natesty2022** [reply]
Ok Thanks :)
